cnchi/installation/install.py

```diff
@@ -774,10 +774,10 @@ def update_pacman_conf():
                     else:
```

Comment:
@lots0logs 

Done.

> 99 characters as that is against Python Coding Standards

That'll be very hard for me to follow, bad habits catching up with me :eyes:.